### PR TITLE
Fix travis on personal github repositories

### PR DESCRIPTION
**Pull request by DrDaveD**
**Description of the Pull Request (PR):**

This change to .travis.yml fixes travis checks on personal github repositories.  This is useful for testing changes before submitting pull requests.

**This fixes or addresses the following GitHub issues:**

None

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [ ] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- [ ] I have tested this PR locally with a `make test`
- [x] This PR is against the project's master branch
- [x] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [x] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +20,10 @@ addons:
       - libseccomp-dev
 
 before_install:
+  # the first 3 lines enable running travis on personal github repositories
+  - cd ../..
+  - if [ ! -d singularityware ]; then mv * singularityware; fi
+  - cd singularityware/singularity
```
Comment (ArangoGutierrez) [inline .travis.yml]: Heya @DrDaveD , could you update this to `sylabs/singularity` ;) 
to update to recent changes.

Comment (DrDaveD) [reply .travis.yml]: Done

**Comment by DrDaveD** [comment]
@ArangoGutierrez can this be approved now?

**Review by ArangoGutierrez — APPROVED** [review]
Works for me. 
(cherry picked the commit into my master branch and my personal travis worked)

**Comment by ArangoGutierrez** [comment]
@DrDaveD could you post this PR on the `devel` channel @ slack, and ask users to test this. 
I think this PR require community approval.

## How to test/reproduce this PR
create a personal branch with the changes on introduced on this PR, on a personal branch, go into[ Travis UI](https://travis-ci.org/), and enable your singularity fork. and wait.... 10 mins.

**Comment by DrDaveD** [comment]
Isn't this a ridiculously trivial change?  I don't understand the reluctance.

**Comment by jscook2345** [comment]
This looks reasonable to me

**Review by jscook2345 — APPROVED** [review]

**Comment by ArangoGutierrez** [comment]
@DrDaveD , 2 approvals, once Travis is happy we can proceed

**Pull request merged**